supply_energy.md

```diff
@@ -0,0 +1,291 @@
+- [X] Coal w/ CCS
+- [X] Gas w/o CCS
+- [X] Gas w/ CCS
+- [X] Oil w/o CCS
+- [X] Oil w/ CCS
+- [X] Bioenergy w/o CCS
+- [X] Bioenergy w/ CCS
+- [X] Geothermal power
+- [X] Nuclear power
+- [X] Solar power
+- [X] Solar power-central PV
+- [X] Solar power-distributed PV
+- [X] Solar power-CSP
+- [X] Wind power
+- [X] Wind power-onshore
+- [X] Wind power-offshore
+- [X] Hydroelectric power
+- [ ] Ocean power
+
+Hydrogen production
+- [X] Coal to hydrogen w/o CCS
+- [X] Coal to hydrogen w/ CCS
+- [X] Natural gas to hydrogen w/o CCS
+- [X] Natural gas to hydrogen w/ CCS
+- [X] Oil to hydrogen w/o CCS
+- [X] Oil to hydrogen w/ CCS
+- [X] Biomass to hydrogen w/o CCS
+- [X] Biomass to hydrogen w/ CCS
+- [X] Nuclear thermochemical hydrogen
+- [X] Solar thermochemical hydrogen
+- [X] Electrolysis
+
+Refined liquids
+- [X] Coal to liquids w/o CCS
+- [X] Coal to liquids w/ CCS
+- [X] Gas to liquids w/o CCS
+- [X] Gas to liquids w/ CCS
+- [X] Bioliquids w/o CCS
+- [X] Bioliquids w/ CCS
+- [X] Oil refining
+
+Refined gases
+- [X] Coal to gas w/o CCS
+- [X] Coal to gas w/ CCS
+- [X] Oil to gas w/o CCS
+- [X] Oil to gas w/ CCS
+- [X] Biomass to gas w/o CCS
+- [X] Biomass to gas w/ CCS
+
+Heat generation
+- [X] Coal heat
+- [X] Natural gas heat
+- [X] Oil heat
+- [X] Biomass heat
+- [X] Geothermal heat
+- [X] Solarthermal heat
+- [X] CHP (coupled heat and power)
+
+### Grid Infrastructure
+
+Electricity
+- [X] Yes (aggregate)
+- [ ] Yes (spatially explicit)
+
+Gas
+- [X] Yes (aggregate)
+- [ ] Yes (spatially explicit)
+
+Heat
+- [X] Yes (aggregate)
+- [ ] Yes (spatially explicit)
+
+CO2
+- [X] Yes (aggregate)
+- [ ] Yes (spatially explicit)
+
+Hydrogen
+- [X] Yes (aggregate)
+- [ ] Yes (spatially explicit)
+
+<a name="table_footnote">1</a>: Note that this table differs from the one provided on the [Supply Inputs Page](inputs_supply.html#description) in that it lists all inputs to the land model, including information passed from other modules. Additionally, the units listed are the units GCAM requires, rather than the units the raw input data uses.
+
+## References
+
+<a name="deloitte2008">[Deloitte 2008]</a> Deloitte Development LLC, 2008, *Geothermal Risk Mitigation Strategies Report*, prepared for Department of Energy, Office of Energy Efficiency and Renewable Energy, Geothermal Program. [Link](https://www1.eere.energy.gov/geothermal/pdfs/geothermal_risk_mitigation.pdf)
+
+<a name="dooley2009">[Dooley and Dahowski 2009]</a> Dooley, J.J., and Dahowski, R.T. 2009. Large-scale U.S. unconventional fuels production and the role of carbon dioxide capture and storage technologies in reducing their greenhouse gas emissions. *Energy Procedia* 1(1), pp. 4225-4232. [Link](http://www.sciencedirect.com/science/article/pii/S1876610209008765)
+
+<a name="enerdata2014">[Enerdata 2014]</a> Enerdata, 2016. *The Future of Gas-to-Liquid (GTL) Industry*. [Link](http://www.enerdata.net/enerdatauk/press-and-publication/energy-news-001/future-gas-liquid-gtl-industry_29879.html)
+
```

Comment:
add:

[Eurek et al. 2017] Eurek, K., P. Sullivan, M. Gleason, D. Hettinger, D. Heimiller, A. Lopez (2017). An improved global wind resource estimate for integrated assessment models. Energy Economics, 64, Link